pkg/scanner/scan.go

```diff
@@ -16,8 +16,8 @@ import (
 	"golang.org/x/xerrors"
 )
 
-func ScanImage(imageName, filePath string) (map[string][]vulnerability.DetectedVulnerability, error) {
-	var err error
+func ScanImage(imageName, filePath string, scanOptions types.ScanOptions) (map[string][]vulnerability.DetectedVulnerability, error) {
+	// var err error
```

Comment:
unnecessary?

Reply:
I have build errors when keeping this line : 
```$ go build cmd/trivy/main.go 
# github.com/knqyf263/trivy/pkg/scanner
pkg/scanner/scan.go:20:6: err declared and not used``` 
I forgot to un-comment before commit

Reply:
I have uncommented it

Reply:
You can remove this line.

Reply:
You used `if` block, so the scope has been changed. As a result, this line is already unnecessary.

Reply:
Done